packages/react-core/src/components/Page/examples/CenteredSection.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Page,
+  Masthead,
+  MastheadToggle,
+  MastheadMain,
+  MastheadBrand,
+  MastheadContent,
+  PageSidebar,
+  PageSection,
+  PageToggleButton,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Card,
+  CardBody
+} from '@patternfly/react-core';
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
+
+export const CenterAlignedPageSection: React.FunctionComponent = () => {
+  const [isNavOpen, setIsNavOpen] = React.useState(true);
+
+  const onNavToggle = () => {
+    setIsNavOpen(!isNavOpen);
+  };
+  const headerToolbar = (
+    <Toolbar id="centered-toolbar">
+      <ToolbarContent>
+        <ToolbarItem>header-tools</ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  const Header = (
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton
+          variant="plain"
+          aria-label="Global navigation"
+          isNavOpen={isNavOpen}
+          onNavToggle={onNavToggle}
+          id="centered-nav-toggle"
+        >
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand href="https://patternfly.org" target="_blank">
+          Logo
+        </MastheadBrand>
+      </MastheadMain>
+      <MastheadContent>{headerToolbar}</MastheadContent>
+    </Masthead>
+  );
+  const Sidebar = <PageSidebar nav="Navigation" isNavOpen={isNavOpen} id="main-padding-sidebar" />;
```

Comment:
I'd add newlines around const declarations (where missing) to make the code easier to read. However I don't have strict opinion on this, what do you think @nicolethoen?

```suggestion

  const Sidebar = <PageSidebar nav="Navigation" isNavOpen={isNavOpen} id="main-padding-sidebar" />;
  
```

Reply:
I don't have a particularly strong opinion here. I think there are definitely times that using line breaks to clearly group related concepts or separate distinct concepts/steps can help with readability. I think in this case, @Dominik-Petrik can make a judgement call if he thinks adding the space is an improvement.